packages/seal/src/client.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import type { Signer } from '@mysten/sui/dist/cjs/cryptography/keypair.js';
```

Comment:
@mysten/sui/cryptography
```suggestion
import type { Signer } from '@mysten/sui/cryptography';
```